package.json

```diff
@@ -35,6 +35,19 @@
     "test": "jest",
     "types": "tsc --noEmit"
   },
+  "husky": {
```

Comment:
Can we remove these changes from this PR? I'd like to separate out the lint-staged changes, since we have that in another PR already.